Automattic/wp-calypso
# Build: add script to run Jetpack cloud concurrently to Calypso in development

#### Changes proposed in this Pull Request

With the recent WordPress.com and Jetpack cloud alignment initiative, writing code that must run on both platforms has become common. For UI development specifically, developers need to be able to see and play with both platforms simultaneously.
This PR allows to run Jetpack cloud on port 3001 while Calypso is running on port 3000. Since both builds have a lot of commonalities, starting Jetpack cloud after Calypso should be quite fast. 

#### Implementation notes

The new `start-jetpack-cloud-p` script doesn't run the steps that are already part of the Calypso build.

I don't have an in-depth knowledge of the build chain so I'm eager to know if there's a better way to do this or if something is not working in your environment.

#### Testing instructions

1. Download the PR
2. Run Calypso with `yarn start`
3. Once it's ready, check that you can navigate through Calypso on port 3000
4. Run cloud concurrently with `yarn run start-jetpack-cloud-p`
5. Once it's ready,  check that you can navigate through Jetpack cloud on port 3001
6. Check that an update in the interface will be reflected on both instances after refreshing
